scripts/vsc-venv-wrapper/vsc-venv.sh

```diff
@@ -0,0 +1,182 @@
+SCRIPT_NAME=${BASH_SOURCE[0]} # $0 cannot be used as it gives '-bash' when sourced
+
+usage() {
+  echo "Usage: source $SCRIPT_NAME {activate <requirements.txt> [modules.sh] | deactivate}"
+  echo ""
+  echo "Commands:"
+  echo "  activate <requirements_file> [modules_file] Activate the environment using the specified requirements file."
+  echo "                                              Optionally, provide a text file with a list of modules to load."
+  echo ""
+  echo "  deactivate                                  Deactivate the virtual environment."
+  echo ""
+  echo "Example Usage:"
+  echo "  $ source $SCRIPT_NAME activate requirements.txt modules.txt"
+  echo "  $ python my_script.py"
+  echo "  $ source $SCRIPT_NAME deactivate"
+  return 1
+}
+
+echo_info() { echo -e "\e[32m[INFO] $1\e[0m"; }
+echo_warning() { echo -e "\e[33m[WARNING] $1\e[0m"; }
+echo_error() { echo -e "\e[31m[ERROR] $1\e[0m"; }
+
+load_modules() {
+  local modules_file lines
+  modules_file="$1"
+
+  if ! mapfile -t lines < "$modules_file"; then # Read the file line by line into an array
+    echo_error "Could not read modules script '$modules_file'"
+    return 1
+  fi
+
+  for line in "${lines[@]}"; do # Loop over each line in the array
+
+    # Skip empty lines
+    if [[ -z "$line" ]]; then
+      continue
+    fi
+
+    if module load "$line"; then
```

Comment:
Remove the quotes, so that it works fine if multiple module names are listed on 1 line in `modules.txt`:
```suggestion
    if module load $line; then